lib/log.bash

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+export ERROR_LOG_LEVEL=1
+export WARNING_LOG_LEVEL=2
+export ALL_LOG_LEVEL=3
+
+function _log_debug()
+{
+  _about 'log a debug message by echoing to the screen. needs LOG_LEVEL >= ALL_LOG_LEVEL'
+  _param '1: message to log'
+  _example '$ _log_debug "Loading plugin git..."'
+  _group 'log'
+
+  [[ "$LOG_LEVEL" -ge $ALL_LOG_LEVEL ]] || return
+  echo -e "${echo_green}$1"
```

Comment:
You need to add an `${echo_normal}` after the `$1`, otherwise the green color stays active.

Reply:
done